lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -0,0 +1,68 @@
+# Tutorial: Configuring a Lambda function to access Amazon RDS in an Amazon VPC
+#   https://docs.aws.amazon.com/lambda/latest/dg/services-rds-tutorial.html
+# https://www.psycopg.org/docs/usage.html
+
+import logging
+import os
+import psycopg2
+import sys
+
+OPT_IN_QUERY = """SELECT va_profile_opt_in(%s, %s, %s, %s);"""
+OPT_OUT_QUERY = """SELECT va_profile_opt_out(%s, %s, %s, %s);"""
+
+NOTIFICATION_API_DB_URI = os.getenv("notification_api_db_uri")
+
+if NOTIFICATION_API_DB_URI is None:
+    logging.error("The database URI is not set.")
+    sys.exit("Couldn't connect to the database.")
+
+# https://www.psycopg.org/docs/module.html#exceptions
+try:
+    connection = psycopg2.connect(NOTIFICATION_API_DB_URI)
+except psycopg2.Warning as e:
+    logging.warning(e)
+except psycopg2.Error as e:
+    logging.exception(e)
+
+    # https://www.psycopg.org/docs/errorcodes.html
+    sys.exit(e.pgcode)
+
+
+def va_profile_opt_in_out_lambda_handler(event: dict, context) -> dict:
+    """
+    Use the event data to process veterans' opt-in/out requests as relayed by VA Profile.  The data fields
+    are as specified in the "VA Profile Syncronization" document.
+        https://docs.aws.amazon.com/lambda/latest/dg/python-handler.html
+    """
+
+    logging.debug(event)
+
+    response = {
+        "statusCode": 200,
+        "headers": {
+            "Content-Type": "application/json",
+        },
+        "body": "Success",
+    }
+
+    try:
+        params = (                                    # Stored function parameters:
+            event["bios"]["VaProfileId"],             #     _va_profile_id
+            event["bios"]["CommunicationItemId"],     #     _communication_item_id
+            event["bios"]["CommunicationChannelId"],  #     _communication_channel_name
+            event["bios"]["confirmationDate"],        #     _timestamp
+        )
+
+        # Execute the appropriate stored function.
+        with connection.cursor() as c:
+            c.execute(OPT_IN_QUERY if event["bios"]["allowed"] else OPT_OUT_QUERY, params)
```

Comment:
This might be wrong; I'm not sure of the type for `event["bios"]["allowed"]`.  If it is "true" or "false" (as strings), rather than a boolean, it will always be True because `bool(<non-empty string>) --> True`.